pkgs/by-name/cw/cwe-client-cli/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, rustPlatform, fetchFromGitHub, webkitgtk, gtk3, cairo, gdk-pixbuf
+, pkg-config, glib, dbus, openssl_3, librsvg, libsoup }:
+let
+  libraries = [
+    webkitgtk
+    gtk3
+    cairo
+    gdk-pixbuf
+    pkg-config
+    glib
+    dbus
+    openssl_3
+    librsvg
+    libsoup
+  ];
+in rustPlatform.buildRustPackage {
+  pname = "cwe-client-cli";
+  version = "0.2.0";
+  buildInputs = libraries;
+  nativeBuildInputs = libraries;
+
+  src = fetchFromGitHub {
+    owner = "NotBalds";
+    repo = "cwe-client-cli";
+    rev = "b4b2bce9bef61aeb1c1c36586503c265e19ed7ad";
+    hash = "sha256-rxJPTqOlKK3V25iE3H7FyEDkrDMANq4HKVvkG62z6W4=";
+  };
+
+  cargoSha256 = "sha256-VyeN3gzvZDIC7wzGhi9t9B9n/rIYLZ8MA6RLcHD/ZyI=";
+
+  meta = {
+    description = "Simple command line client for CWE";
+    homepage = "https://github.com/NotBalds/cwe-client-cli";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ tbwanderer ];
+    mainProgram = "cwe-client-cli";
```

Comment:
```suggestion
    mainProgram = "cwe-client-cli";
    platforms = lib.platforms.x86_64;
```
Add a list of platforms this can build on, omitting the platforms where this can't be used. From the [.SRCINFO](https://github.com/NotBalds/cwe-client-cli/blob/242bdde883f852c49051fb32af2be8789205901f/.SRCINFO#L5), I gathered this is x86-only, but you can use `lib.platforms.linux` to only build for linux targets of various architectures, or `lib.platforms.unix` to build for all UNIX targets of various architectures, including AArch64 macOS. See [doubles.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix) for a complete list of common lists, or define the platforms manually with a list of string.